vyper/venom/ir_node_to_venom.py

```diff
@@ -127,8 +135,7 @@ def _convert_binary_op(
     swap: bool = False,
 ) -> Optional[IRVariable]:
     ir_args = ir.args[::-1] if swap else ir.args
-    arg_0 = _convert_ir_basicblock(ctx, ir_args[0], symbols, variables, allocated_variables)
-    arg_1 = _convert_ir_basicblock(ctx, ir_args[1], symbols, variables, allocated_variables)
+    arg_0, arg_1 = _convert_ir_bb_list(ctx, ir_args, symbols, variables, allocated_variables)
 
     return ctx.get_basic_block().append_instruction(str(ir.value), arg_1, arg_0)
```

Comment:
this is a mypy hint right? can we do
```suggestion
    assert isinstance(ir.value, str)  # mypy hint
    return ctx.get_basic_block().append_instruction(ir.value, arg_1, arg_0)
```